<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="FileMaskList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.txt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.doc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.png</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.wav</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.mp3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.html</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.js</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.css</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.bmp</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
